mongo/session.go

```diff
@@ -76,8 +76,9 @@ func SessionFromContext(ctx context.Context) Session {
 // Session is an interface that represents a MongoDB logical session. Sessions can be used to enable causal consistency
 // for a group of operations or to execute operations in an ACID transaction. A new Session can be created from a Client
 // instance. A Session created from a Client must only be used to execute operations using that Client or a Database or
-// Collection created from that Client. Custom implementations of this interface should not be used in production. For
-// more information about sessions, and their use cases, see https://www.mongodb.com/docs/manual/reference/server-sessions/,
+// Collection created from that Client. Instances of Session are not thread safe or fork safe. Custom implementations of
+// this interface should not be used in production. For  more information about sessions, and their use cases, see
```

Comment:
Prefer Go-specific terminology for describing concurrency to prevent confusion. Also, since `Session` is an interface, it's more correct to say "implementations" rather than "instances".

For example:
> Implementations of Session are not safe for concurrent use by multiple goroutines.